Gemfile

```diff
@@ -6,7 +6,8 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'cocoapods'
+  gem 'cocoapods', git: 'https://github.com/CocoaPods/CocoaPods.git'
+  gem 'cocoapods-core', git: 'https://github.com/CocoaPods/Core.git'
```

Comment:
unpin from git before merging

Reply:
we probably won't have a cocoapods release out with the changes on master for at least a month